src/include/com/amazonaws/kinesis/video/common/Include.h

```diff
@@ -102,6 +102,11 @@ extern "C" {
  */
 #define MAX_REGION_NAME_LEN 128
 
+/**
+ * Maximum allowed service name length
+ */
+#define MAX_SERVICE_NAME_LEN 32

```

Comment:
Perhaps someone from AWS can comment on the max length?

Reply:
I went ahead and made this `128` - there is some precedent as that is what max region len is. I have also never seen an aws service even close to this length, so am confident it is safe